conf/config.yaml

```diff
@@ -146,3 +146,11 @@ plugins:                          # plugin list
 
 stream_plugins:
   - mqtt-proxy
+
+eureka:
+  urls:
```

Comment:
please the style of etcd

Reply:
What is style of etcd? Give me an example， please

Reply:
please check `conf/config.yaml` by yourself.

Reply:
@moonming  done, review please.